### ISSUE TITLE
Broadlink send command

### ISSUE THREAD

**Issue by kaidgreat** [TomerFi/home_assistant_custom_ac_alexa_smarthome_skill#6]
Hi Tomer,

Am really noob at programming and new to HA.. i managed to understand how the code works in laymans term. Now, am struggling to get the broadlink to work. When i add the broadlink platform, i only see broadlink.send in the services. The switch.broadlink.send doesnt appear.. is this new for HA or am i doing wrong?

**Comment by TomerFi** [comment]
It's not you...
In release 0.92.0 there was a breaking change moving the broadlink service from the switch component to the broadlink component.
And on top of that the service has changed name and arguments.

I've just now upgraded from 0.88.1 to 0.93.0 so I wasn't even aware of the change.
It's already fixed in my environment, I'll do my best to fix the repository as soon as I can.

**Comment by kaidgreat** [comment]
Thanks Tomer for the reply. I thought there was something wrong with my broadlink switch. Hopefully you can fix the repo. :)

**Comment by kaidgreat** [comment]
Hello Tomer, i managed to resolve the broadlink.send issue.. i can already control the ac through HA. Now my issue is that amazon couldnt login to my HA. I read that api_password no longer work with newer ver. Of HA. Anyway you can help on this issue?  Thanks

**Comment by TomerFi** [comment]
Yeah that won't work.
This repository is no longer maintained.

There are better solutions for this, you can find it [my home assistant configuration](https://github.com/TomerFi/my_home_assistant_configuration) and [my appdaemon configuration](https://github.com/TomerFi/my_appdaemon_configuration).

Please feel free to continue the conversation if needed even thought the issue is closed.
